pkg/stanza/operator/parser/container/parser_test.go

```diff
@@ -157,16 +157,18 @@ func TestProcess(t *testing.T) {
 			},
 			&entry.Entry{
 				Attributes: map[string]any{
-					"time":                        "2029-03-30T08:31:20.545192187Z",
-					"log.iostream":                "stdout",
+					"time":          "2029-03-30T08:31:20.545192187Z",
```

Comment:
I know it's not from this PR, but any reason to keep original timestamp after we parse it?

Reply:
I think the original approach was preserving the original time after parsing. The `time_parser` does not remove the original time field from what I can see in its [docs](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/pkg/stanza/docs/types/timestamp.md#parse-a-timestamp-using-a-strptime-layout).
So the question is if we want to follow the same approach here for compatibility reasons for those users that might still use these for any reason.
Based on the above, leaving it as is and discussing (in follow-up issue) its removal through an optional setting might be the safest path?

Reply:
Makes sense. I am in favor of removing it after parsing, but a followup PR makes sense.